snap/quota/resources_test.go

```diff
@@ -60,3 +73,17 @@ func (s *resourcesTestSuite) TestQuotaChangeValidation(c *C) {
 		c.Check(err, ErrorMatches, t.err)
 	}
 }
+
+func (s *resourcesTestSuite) TestQuotaChangeValidationPasses(c *C) {
+	tests := []struct {
+		limits       quota.Resources
+		updateLimits quota.Resources
+	}{
+		{quota.NewResources(quantity.SizeMiB), quota.NewResources(quantity.SizeGiB)},
+	}
+
+	for _, t := range tests {
+		err := t.limits.Change(t.updateLimits)
+		c.Check(err, IsNil)
```

Comment:
I would maybe consider checking `c.Check(t.limits, DeepEquals, tc.udpateLimits)` here to ensure that the change really happend?

Reply:
This is a really good point, thanks for the review @mvo5